### PR TITLE
Morse Decoder

### PR DESCRIPTION
**Pull request by BenjaminSemah**
### In this branch, we have done the following

Created a morse decoder using Ruby.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,48 @@
...
+Feel free to check the [issues page](https://github.com/ahmedtaa/morse-decode/issues).
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
+## Acknowledgments
+
+- Microverse
+
```
Comment (mikethreels) [inline README.md]: - [x] the link to the MIT license is not working as intended, please have another look at the [Microverse template](https://github.com/microverseinc/readme-template) for an example of how it is expected to work. please adjust it accordingly.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (BenjaminSemah) [reply README.md]: RESOLVED: Readme file has been updated to include correct link to MIT License.

**Comment by BenjaminSemah** [comment]
Review Implemented: Readme file has been updated to include correct link to MIT License.

**Review by Kingstalux — APPROVED** [review]
Hi @BenjaminSemah and @ahmedtaa,
# Status: Approved ✔️ ✔️ ✔️ 
 
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966

Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**